assets/templates/login.html

```diff
@@ -1,46 +1,68 @@
+        </symbol>
+      </defs>
+    </svg>
+    <div role="application">
+      <main>
+        <div class="login auth">
+          <div class="main-wrapper">
+            <header>
+              <figure>
+                <div class="svg-wrapper">
+                  <svg>
+                    <use xlink:href="#cozy-icon" />
+                  </svg>
+                </div>
+              </figure>
+              {{if .PublicName}}
+                <h1>Welcome {{.PublicName}}!</h1>
+              {{else}}
+                <h1>Welcome!</h1>
+              {{end}}
+            </header>
             <div role="region">
-              <h1>Welcome</h1>
-              <p id="login-passphrase-tip" class="help">Please enter your passphrase to access to your Cozy</p>
-              <label for="login-passphrase" aria-describedby="login-passphrase-tip" class="with-input">
-                <input id="login-passphrase" name="passphrase" placeholder="Passphrase" autofocus="" type="password">
-              </label>
+              <form id="login-form" method="POST" action="/auth/login" class="login auth">
+                <input type="hidden" name="redirect" value="{{.Redirect}}" />
+                <p class="help" id="login-password-tip">Enter your password to access your Cozy</p>
```

Comment:
After discussion with @kelukelu , @pierrevdk and @jsilvestre , we've decided to keep 'password' for user. 'Passphrase' seems too much technical.